src/store.js

```diff
@@ -41,16 +55,22 @@ const store = new Vuex.Store({
     zipcodesReceived(state, zipcodes) {
       state.zipcodes = zipcodes;
     },
-    filtersReceived(state, filters) {
-      state.filters = filters;
-    },
     viewToggled(state) {
       state.view = state.view === 'map' ? 'list' : 'map';
+    },
+    setFilters(state, filters) {
+      state.filters = {...state.filters, ...filters}
+    },
+  },
+  getters: {
```

Comment:
Hmm, yeah I always assumed they were, but now that I've re-read docs I don't see it called out explicitly so I could be mistaken.